sdarq/deploy.sh

```diff
@@ -24,6 +24,7 @@ export SDARQ_SECRET="sdarq"
 export CIS_JOB_TOPIC="cis-scans"
 export ZAP_JOB_TOPIC="zap-scans"
 export SC_FIRESTORE_COLLECTION="security-controls"
+export SAST_FIRESTORE_COLLECTION="sast-details"
```

Comment:
Why `SAST_FIRESTORE_COLLECTION` is added to SDARQ? SDARQ's `frontend` or `backend` is not using that Firestore collection.

Reply:
It's there for reference in Firestore directly for now; we can expose this in the UI or reference it in policy enforcement code like the slack alerts as well.

Reply:
But we are not using it in the Sdarq namespace, if we are going to implement it in the future, we can add it.

Reply:
Removing from sdarq for now.